addons/website/static/src/scss/primary_variables.scss

```diff
@@ -2173,6 +2173,14 @@ $o-theme-font-configs: (
         'family': ('Arvo', Times, serif),
         'url': 'Arvo:300,300i,400,400i,700,700i',
     ),
+    'Figtree': (
+        'family': ('Figtree', sans-serif),
+        'url': 'Figtree:300,300i,400,400i,500,500i,700,700i',
+    ),
+    'Inter': (
+        'family': ('Inter', sans-serif),
+        'url': 'Inter:300,300i,400,400i,700,700i',
+    ),
```

Comment:
Ok. FYI, the difference between "normal" and "bold" it's still extremely light when the `lead` class is applied, but the issue is not related to your changes, it should be addressed in another task.

![image](https://github.com/odoo/odoo/assets/8558351/baf70a53-81e0-4005-b89e-ab9f2aaaa53d)